src/omotes_simulator_core/entities/network_controller.py

```diff
@@ -44,6 +44,23 @@ def __init__(
         self.producers = producers
         self.consumers = consumers
         self.storages = storages
+        self._set_priority_from_marginal_costs()
+
+    def _set_priority_from_marginal_costs(self) -> None:
+        """This method sets the priority of the producers based on the marginal costs."""
+        # Sort the producers based on the marginal costs
+        self.producers.sort(key=lambda x: x.marginal_costs)
```

Comment:
This will sort all the assets based on marginal cost, but when the values are zero (e.g., no cost information available) the assets will have 'top' priority right? 

![alt text](image.png)

Reply:
indeed, need to discussed if this is desired behaviour